crates/phonon/src/audio_buffer.rs

```diff
@@ -81,26 +108,26 @@ impl<const N_CHANNELS: usize, const N_SAMPLES: usize> AudioBuffer<N_CHANNELS, N_
                 sum += self[j][i];
             }
 
-            other[0][i] = sum * factor;
+            output[0][i] = sum * factor;
         }
     }
 
-    /// Writes interleaved slice to `AudioBuffer`.
-    /// todo: Check perf?
-    pub fn write(&mut self, other: &[f32; N_CHANNELS * N_SAMPLES]) {
+    /// Reads a slice of interleaved samples into an `AudioBuffer`.
```

Comment:
Or maybe "this `AudioBuffer`?
```suggestion
    /// Reads a slice of interleaved samples into the `AudioBuffer`.
```